index.js

```diff
@@ -58,7 +65,7 @@ class PdfTk {
              * @returns {String} Path of the newly created temp file.
              */
             const writeTempFile = srcFile => {
-                const tmpPath = path.join(__dirname, './node-pdftk-tmp/');
+                const tmpPath = path.join(__dirname, this._tempDir);
```

Comment:
You don't actually want to make it relative here. Perhaps you meant

`const tmpPath = this._tempDir`